src/app/api/repositories/[owner]/[repo]/activate/route.ts

```
@@ -1,49 +1,14 @@
 export async function PUT(
   request: Request,
-  { params }: { params: { owner: string; repo: string } }
+  context: { params: { owner: string; repo: string } }
 ) {
-  const session = await getServerSession(authOptions);
-  if (!session?.user?.email) {
-    return new NextResponse("Unauthorized", { status: 401 });
-  }
+  const { session } = await validateSession();
+  const { owner, repo } = await context.params;
+  const [userId, repository] = [session.user?.id!, `${owner}/${repo}`];
```

ESLint

[error] Optional chain expressions can return undefined by design - using a non-null assertion is unsafe and wrong.

(@typescript-eslint/no-non-null-asserted-optional-chain)